src/redux/rootReducer.js

```diff
@@ -52,5 +54,7 @@ export const rootReducer = combineReducers({
   InformationModelsReducer,
   OrganizationsCatalogReducer,
   PublicServicesReducer,
-  PublicServiceReducer
+  PublicServiceReducer,
+  PublicServicesRequiredByReducer,
+  PublicServicesRelatedByReducer
```

Comment:
Tenker at det hadde vært fint å plassere funksjonalitet i `PublicServicesRequiredByReducer` og `PublicServicesRelatedByReducer ` under `PublicServicesReducer `, fordi det er samme underliggende modell (`PublicService[]`). For å kunne skille de, så er det mulig å lage en dedikert nøkkel (`requiredByPublicServices` og `relatedByPublicServices` eller tilsvarende)

Reply:
Enig, det er bedre. Oppdatert PR.